src/strings/unicode_ops.c

```diff
@@ -108,13 +108,41 @@ MVMGrapheme32 MVM_unicode_lookup_by_name(MVMThreadContext *tc, MVMString *name)
     MVMuint64 size;
     char *cname = MVM_string_ascii_encode(tc, name, &size, 0);
     MVMUnicodeNameRegistry *result;
+
     if (!codepoints_by_name) {
         generate_codepoints_by_name(tc);
     }
     HASH_FIND(hash_handle, codepoints_by_name, cname, strlen((const char *)cname), result);
     MVM_free(cname);
+
     return result ? result->codepoint : -1;
 }
+/* XXX move this so it's included properly */
+MVMString * MVM_string_from_grapheme(MVMThreadContext *tc, MVMCodepoint cp) {
+    MVMString *s;
+    MVMGrapheme32 g;
+    MVMNormalizer norm;
+
+    MVM_unicode_normalizer_init(tc, &norm, MVM_NORMALIZE_NFG);
+    if (!MVM_unicode_normalizer_process_codepoint_to_grapheme(tc, &norm, cp, &g)) {
+        MVM_unicode_normalizer_eof(tc, &norm);
+        g = MVM_unicode_normalizer_get_grapheme(tc, &norm);
```

Comment:
Though as mentioned below, this whole pass through the normalizer may not make sense.